packages/actor-query-operation-filter-sparqlee/test/ActorQueryOperationFilterSparqlee-test.ts

```diff
@@ -2,14 +2,32 @@
 import { literal, variable } from "@rdfjs/data-model";
 import { ArrayIterator } from "asynciterator";
 import { Map } from "immutable";
-import { Algebra, Factory } from "sparqlalgebrajs";
+import { Algebra, Factory, translate } from "sparqlalgebrajs";
 import * as sparqlee from "sparqlee";
 const arrayifyStream = require('arrayify-stream');
 
+import { KEY_CONTEXT_BASEIRI } from "@comunica/actor-init-sparql/index-browser";
```

Comment:
This may resolve the drop in coverage:
```suggestion
import { KEY_CONTEXT_BASEIRI } from "@comunica/actor-init-sparql";
```